TheMovieDb/Modules/Home/MovieFeedCell.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  MovieFeedCell.swift
+//  TheMovieDb
+//
+//  Created by Ricardo Ramirez on 29/10/21.
+//
+
+import UIKit
+import Kingfisher
+
+class MoviesFeedCell: UICollectionViewCell {
+    
+    @IBOutlet weak var poster: UIImageView!
+    
+    func updateUI(withMovie movie: Movie) {
+        if let posterpath = movie.posterPath,
+           let posterURL = URL(string: "https://image.tmdb.org/t/p/w500/\(posterpath)") {
```

Comment:
Let's make a constants file for strings like  "https://image.tmdb.org/t/p/w500/"